### PR TITLE
Update pages structure

### PR DESCRIPTION
**Pull request by davelange**
- Update pages file structure: create directories for each route. Move `Dashboard` and `Onboarding` components into this structure.
- Add a routes helper, to avoid using string paths everywhere.

### CODE REVIEW

File: renderer/src/pages/dashboard/Dashboard.tsx
Comment (davelange) [inline renderer/src/pages/dashboard/Dashboard.tsx:8]: @juliangruber it would simplify things if we updated `tsconfig` to use absolute paths for imports instead of relative. I can add this in a follow up PR, or add it here if you think it makes sense.

**Review by juliangruber — APPROVED** [review]
Lgtm!

**Comment by juliangruber** [comment]
@bajtos is out over the weekend. I suggest we merge once we get his approval. If you would already like to base new work on top of this one, please create a PR that targets `subvisual:dl/update-routes-structure` as the base branch.

File: renderer/src/pages/dashboard/Dashboard.tsx
Comment (davelange) [reply renderer/src/pages/dashboard/Dashboard.tsx:8]: It's here: https://github.com/subvisual/desktop/pull/1

**Comment by juliangruber** [comment]
@davelange we can go ahead without @bajtos's review

**Pull request merged**
